x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/alert_details_overview.tsx

```diff
@@ -0,0 +1,338 @@
+        description: 'TODO',
+      },
+      {
+        title: 'File Accessed',
+        description: 'TODO',
+      },
+      {
+        title: 'Signer',
+        description: 'TODO',
+      },
+      {
+        title: 'Owner',
+        description: 'TODO',
+      },
+    ];
+  }, []);
+
+  const sourceProcessDetailsColumns = useMemo(() => {
+    return [
+      {
+        title: 'Process ID',
+        description: 'TODO',
+      },
+      {
+        title: 'Process Name',
+        description: 'TODO',
+      },
+      {
+        title: 'Process Path',
+        description: 'TODO',
+      },
+      {
+        title: 'MD5',
+        description: 'TODO',
+      },
+      {
+        title: 'SHA1',
+        description: 'TODO',
+      },
+      {
+        title: 'SHA256',
+        description: 'TODO',
+      },
+      {
+        title: 'MalwareScore',
+        description: 'TODO',
+      },
+      {
+        title: 'Parent Process ID',
+        description: 'TODO',
+      },
+      {
+        title: 'Signer',
+        description: 'TODO',
+      },
+      {
+        title: 'Username',
+        description: 'TODO',
+      },
+      {
+        title: 'Domain',
+        description: 'TODO',
+      },
+    ];
+  }, []);
+
+  const sourceProcessTokenDetailsColumns = useMemo(() => {
+    return [
+      {
+        title: 'SID',
+        description: 'TODO',
+      },
+      {
+        title: 'Integrity Level',
+        description: 'TODO',
+      },
+      {
+        title: 'Privileges',
+        description: 'TODO',
+      },
+    ];
+  }, []);
+
+  return (
+    <>
+      {/* Hard coded top level alert details component. TODO: maybe abstract somewhere else? */}
+      <EuiTitle size="s">
+        <h3>Detected Malicious File</h3>
+      </EuiTitle>
+      <EuiSpacer />
+      <EuiText>
+        <p>
+          Endgame MalwareScore detected the opening of a document with a blah blah blah on{' '}
+          {alertDetailsData.host.hostname} on {AlertDetailsDate}
+        </p>
+      </EuiText>
+      <EuiSpacer />
+      <EuiText>
+        Endpoint Status: <EuiHealth color="success">Online</EuiHealth>
+      </EuiText>
+      <EuiSpacer />
+      <EuiText>Alert Status: Open</EuiText>
+      <EuiSpacer />
+
+      {/* Start of Alert Details overview component TODO: delete this comment eventually */}
+      <EuiAccordion
+        id="alertDetailsAlertAccordion"
+        buttonContent="Alert"
+        paddingSize="l"
+        initialIsOpen={true}
+      >
+        <EuiDescriptionList
+          type="column"
+          listItems={alertDetailsColumns}
+          style={{ maxWidth: '400px' }}
+        />
+      </EuiAccordion>
+
+      <EuiSpacer />
+
+      <EuiAccordion
+        id="alertDetailsHostAccordion"
+        buttonContent="Host"
+        paddingSize="l"
+        initialIsOpen={true}
+      >
+        <EuiDescriptionList
+          type="column"
+          listItems={hostDetailsColumns}
+          style={{ maxWidth: '400px' }}
```

Comment:
Could we use https://styled-components.com/ instead of inline styles?

Reply:
this is actually just copied over from the eui example, ill probably end up removing it, but for future styles, yes